02-spatial-data.Rmd

```diff
@@ -24,24 +28,29 @@ library(raster)
-Both are vital to the sciences, although which will be of most use will largely depend on your discipline:
+This chapter will provide brief explanations of the fundamental geographic data models: vector and raster.
+We will introduce briefly the theory behind both data models, and then move quickly towards how these are implemented in R.
+Vector and raster models are vital to geospatial analysis [@longley_geographic_2015].
+The vector data model represents the world using points, lines and polygons.
+Hence, it supports data with discrete, well-defined borders.
+On the other hand, the raster data model is good at representing continous phenomena such as elevation or rainfall with the help of grid cells.
+Naturally, it depends on your discipline if you more often user raster or vector data:
 
-- Because human settlements and boundaries tend to be complex with precise borders defined by legal systems, vector data tends to dominate in the social sciences.
-- In environmental sciences, by contrast, raster data tend to dominate due to its links to remote sensing.
+- Vector data tends to dominate the social sciences because human settlements and boundaries have discrete borders.
+- By contrast, raster data often dominates the environmental sciences because these often use remotely sensed imagery. 
 
 However, there is a substantial level of overlap:
-ecologists and demographers, for example, commonly use both vector and raster geographical data types.
-We therefore strongly recommend learning about each type of data before proceeding to understand how to manipulate them in subsequent chapters.
+ecologists and demographers, for example, commonly use both vector and raster data.
+We, therefore, strongly recommend learning about each data model before proceeding to understand how to manipulate them in subsequent chapters.
 
 ## Vector data
 
 Vector data are based on points that are located on a Cartesian (or geographic) coordinate system.
-Each point in vector data is typically described by two numbers representing distance from the $origin$ along the $x$ (horizontal) and $y$ (vertical) axis in Euclidean space.^[In 3 dimensional coordinate systems three numbers are needed for each of the three axes, $x$, $y$ and $z$.]
+Typically, a point consists of an x and a y value. 
+These two numbers simply represent the distance from the $origin$ along the $x$ (horizontal) and $y$ (vertical) axis in Euclidean space.^[In 3 dimensional coordinate systems three numbers are needed for each of the three axes, $x$, $y$ and $z$.]
 In mathematical notation these points are typically represented as numbers separated by commas and enclosed by a pair of brackets: 
 $(1, 3)$ for example, represents a point located one unit to the right and three units above the origin.
-There is clear link between these vector points and the `vector` class in R.
+There is a clear link between these vector points and the `vector` class in R.
```

Comment:
Agreed - the previous phrasing risks confusing readers. There is a clear distinction between an abstract data model and a data class implemented in a language such as R.